### ISSUE TITLE
Unable to upload file with Chinese name

### ISSUE THREAD

**Issue by Tajang-ctf** [kingToolbox/WindTerm#611]
After connecting to a Windows machine, uploading files with Chinese names in the file manager panel fails. And the file manager does not display files with Chinese names.
Please see the picture below, when I refresh the file manager panel on the left, the files with Chinese names are still not displayed. But on the right I searched the file list and found that there are these files.
![Snipaste_2022-03-22_19-06-56](https://user-images.githubusercontent.com/63721558/159469210-2b4d6bf7-1ed9-42c8-922b-7bfd0695d5ac.png)

**Comment by kingToolbox** [comment]
Is the Windows server running `OpenSSH` as the `SSH` server? Is it okay if you upload a file with an English filename? Also, browse other folders containing files with Chinese filenames, can you see those files? Thank you.

**Comment by Tajang-ctf** [comment]
Yes, I am using openssh as the ssh server for windows 2012 server R2. I can upload files with English names, but uploading files with Chinese names fails.

**Comment by kingToolbox** [comment]
Ok, does the tooltip of the transfer button show any `Failed Items`? You only need to hover your mouse over the `upload` button in the transfer bar to see it. In subsequent versions, I will add the display of error messages, which will be more convenient.

**Comment by kingToolbox** [comment]
Ok, does the tooltip of the transfer button show any `Failed Items`? You only need to hover your mouse over the `upload` button in the `transfer bar` to see it. In subsequent versions, I will add the display of error messages, which will be more convenient.

**Comment by kingToolbox** [comment]
Ok, does the tooltip of the transfer button show any `Failed Items`? You only need to hover your mouse over the `upload` button in the `transfer bar` to see it. In subsequent versions, I will add the display of error messages, which will be more convenient.
